src/main/resources/docgen/toc.json

```diff
@@ -37,6 +37,20 @@
             {"class":"Doc",  "desc" : "Features Matrix",             "dir" : "", "file": "DistinctCountFeaturesMatrix" },
             {"class":"Doc",  "desc" : "Figures-of-Merit Comparison", "dir" : "", "file": "DistinctCountMeritComparisons"},
             
+            { "class":"Dropdown", "desc" : "CPC Sketches", "array":
+              [
+                {"class":"Doc",  "desc" : "CPC Sketch",                               "dir" : "CPC", "file": "CPC" },
+                { "class":"Dropdown", "desc" : "CPC Examples", "array":
+                  [
+                    {"class":"Doc",  "desc" : "CPC Sketch Java Example",              "dir" : "CPC", "file": "CpcJavaExample" },
+                    {"class":"Doc",  "desc" : "CPC Sketch C++ Example",               "dir" : "CPC", "file": "CpcCppExample" },
+                    {"class":"Doc",  "desc" : "CPC Sketch Pig UDFs",                  "dir" : "CPC", "file": "CpcPigExample" },
+                    {"class":"Doc",  "desc" : "CPC Sketch Hive UDFs",                 "dir" : "CPC", "file": "CpcHiveExample" }
```

Comment:
BTW, commas after the last item in a list is OK.  No need to change now, but FYI.